understory_box_tree/src/tree.rs

```diff
@@ -518,6 +517,29 @@ impl<B: Backend<f64>> Tree<B> {
             filter.matches(node.local.flags)
         })
     }
+
+    /// Iterate live nodes whose world-space bounds contain a world-space point.
+    ///
+    /// Edges of the bounding boxes are included in the contains-check, having the same semantics
+    /// as [`Aabb2D::contains_point`], meaning that a point exactly on the edge of a bounding box
+    /// is contained by that bounding box.
+    ///
+    /// - `point` is interpreted in world coordinates.
+    /// - Nodes must satisfy the [`QueryFilter`] and contain the given point to be yielded.
+    /// - The returned [`NodeId`]s are in an unspecified order; no z-sorting is applied.
+    pub fn containing_point<'a>(
```

Comment:
I'm not entirely happy with the naming of `Tree::containing_point` and `Tree::intersect_rect`. They're grammatically not analogous though they conceptually do the same thing, but at least with this naming it's somewhat clear they both yield numerous nodes (as iterator). In `understory_index` these are called `Backend::{query_point, query_rect}`. Perhaps we can lift those names to here as well.